src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -960,47 +960,54 @@ class AlterTableTableColumnActionSegment(BaseSegment):
             Sequence(
                 "ADD",
                 "COLUMN",
-                Ref("ColumnReferenceSegment"),
-                Ref("DatatypeSegment"),
-                OneOf(
-                    # Default
-                    Sequence(
-                        "DEFAULT",
-                        Ref("ExpressionSegment"),
-                    ),
-                    # Auto-increment/identity column
-                    Sequence(
-                        OneOf(
-                            "AUTOINCREMENT",
-                            "IDENTITY",
-                        ),
-                        OneOf(
-                            # ( <start_num>, <step_num> )
-                            Bracketed(
-                                Ref("NumericLiteralSegment"),
-                                Ref("CommaSegment"),
-                                Ref("NumericLiteralSegment"),
+                Sequence(
```

Comment:
I don’t think you need this Sequence as it only includes one component (the Delimited). Can we remove it?

Reply:
Have made this change (and another unnecessary `Sequence`) so we can include this in 0.10.0.